src/babel-register-esm.js

```diff
@@ -15,10 +15,11 @@ const SUPPORTED_EXTENSIONS = ['.js', '.jsx', '.ts', '.tsx']
  */
 export async function resolve(specifier, context, nextResolve) {
   try {
+    if(path.isAbsolute(specifier)) specifier = pathToFileURL(specifier).toString();
```

Comment:
This doesn't pass prettier, and thus doesn't pass `npm test`. Please make sure you pass NPM test and I will merge this PR. 

Thanks!

Reply:
Hi, thank you for the feedback. I have fixed the formatting and the changes now pass `npm test`.